src/workerd/io/compatibility-date.capnp

```diff
@@ -286,4 +286,9 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # Enables the tunneling of exceptions from a dynamic dispatch callee back into the caller.
   # Previously any uncaught exception in the callee would be returned to the caller as an empty
   # HTTP 500 response.
+
+  sqlBinding @28 :Bool
+      $compatEnableFlag("sql_binding")
```

Comment:
We commonly use the word "binding" to mean specifically things that go into `env`, so this name is a bit confusing in the Workers context. (You just mean generic API bindings.)

I think, though, that we don't really need a new compat flag at all. The new API is not a backwards-incompatible change. So we should just gate it on `workerdExperimental` for now, and remove the gate once we're ready to release it.